modules/i18n/src/models/namespaces/myaccount-ns.ts

```diff
@@ -728,6 +728,8 @@ export interface MyAccountNS {
                 profileImage: string;
                 phoneNumbers: string;
                 userName: string;
+                // Below is to allow all set of SCIM attributes, that are dynamically called
+                [key: string]: string | Record<string, string>;
```

Comment:
I think it will be better if we can add all the SCIM attributes manually here.

Reply:
Fixed by https://github.com/wso2/identity-apps/pull/3763/commits/25d7975004f07604b9d6adb6dd1e18041b8ef502